core-web/libs/edit-content/src/lib/fields/dot-edit-content-relationship-field/dot-edit-content-relationship-field.component.html

```diff
@@ -88,5 +89,6 @@
 </p-table>
 
 <dot-select-existing-content
+    [contentId]="$contentId()"
```

Comment:
what is contentID? inode or identifier?

Reply:
the correct name needs to be contentTypeId.

If we leave it as contentId, when we touch this again we will start looking to see if that is an inode or an identifier, in the end it is really a contentTypeId

Reply:
Fixed!